server/public/model/config.go

```diff
@@ -3078,6 +3078,51 @@ func (s *PluginSettings) SetDefaults(ls LogSettings) {
 	}
 }
 
+type WranglerSettings struct {
+	PermittedWranglerUsers                   []string
+	AllowedEmailDomain                       []string
+	MoveThreadMaxCount                       *int64
+	MoveThreadToAnotherTeamEnable            *bool
+	MoveThreadFromPrivateChannelEnable       *bool
+	MoveThreadFromDirectMessageChannelEnable *bool
+	MoveThreadFromGroupMessageChannelEnable  *bool
+}
+
+func (w *WranglerSettings) SetDefaults() {
+	if w.PermittedWranglerUsers == nil {
+		w.PermittedWranglerUsers = make([]string, 0)
+	}
+	if w.AllowedEmailDomain == nil {
+		w.AllowedEmailDomain = make([]string, 0)
+	}
+	if w.MoveThreadMaxCount == nil {
+		w.MoveThreadMaxCount = NewInt64(100)
+	}
+	if w.MoveThreadToAnotherTeamEnable == nil {
+		w.MoveThreadToAnotherTeamEnable = NewBool(true)
+	}
+	if w.MoveThreadFromPrivateChannelEnable == nil {
+		w.MoveThreadFromPrivateChannelEnable = NewBool(true)
+	}
+	if w.MoveThreadFromDirectMessageChannelEnable == nil {
+		w.MoveThreadFromDirectMessageChannelEnable = NewBool(true)
+	}
+	if w.MoveThreadFromGroupMessageChannelEnable == nil {
+		w.MoveThreadFromGroupMessageChannelEnable = NewBool(true)
```

Comment:
Defaults to true in the wrangler plugin - this config is for moving _out_ of DM/GM so I think that's useful, but 0/5.

Reply:
Just thinking that this was something we had disabled in community so we may want to default it to `false` and then admins can explicitly enable it so they are aware this is possible. @johndavidlugtu any thoughts on this?

Reply:
Because plugins are effectively "opt in" per instance, we can somewhat assume that admins have reviewed the settings for the plugin and disable anything they don't want. 

I do think the ability to move threads out of DMs/GMs is useful but should be disabled by default due to privacy concerns.

As we move it to the product (where everyone will have access) there's a stronger case to disable this by default

Reply:
I've set the 2 settings for GM and DM to false by default. I'd argue given the concerns around privacy, we should apply the same default of `false` to moving out of private channels as well?

Reply:
I'd strongly suggest setting all 4 configuration options to `false` by default for the reasons John mentioned.

Reply:
@nickmisasi good point -- yes that should be default `false` as well.